tasks/main.yml

```diff
@@ -1,13 +1,36 @@
 ---
-- include: "install.packages-{{ansible_os_family}}.yml"
+
+- include: "repo-{{ ansible_os_family }}.yml"
+  when: dnsdist_install_repo
+  tags:
+    - install
+    - repository
+
+- include: install-epel.yml
+  when: ansible_os_family == 'RedHat'
```

Comment:
maybe we should have a `dnsdist_install_epel` variable to support people that have private EPEL-like repos?